app/src/main/java/org/simple/clinic/summary/linkId/LinkIdWithPatientView.kt

```diff
@@ -139,37 +139,6 @@ class LinkIdWithPatientView :
     context.injector<Injector>().inject(this)
   }
 
-  override fun onAttachedToWindow() {
-    super.onAttachedToWindow()
-    delegate.start()
-  }
-
-  override fun onDetachedFromWindow() {
-    delegate.stop()
-    super.onDetachedFromWindow()
-  }
-
-  override fun onSaveInstanceState(): Parcelable {
-    return delegate.onSaveInstanceState(super.onSaveInstanceState())
-  }
-
-  override fun onRestoreInstanceState(state: Parcelable?) {
-    super.onRestoreInstanceState(delegate.onRestoreInstanceState(state))
-  }
-
-  @SuppressLint("CheckResult")
-  override fun onFinishInflate() {
-    super.onFinishInflate()
-
-    if (isInEditMode) {
-      return
-    }
-
-    backgroundView.setOnClickListener {
-      // Intentionally done to swallow click events.
-    }
```

Comment:
This is being handled automatically by `BaseBottomSheet`, right? I don't remember if we did this. 😅

Reply:
Yep, we are not allowing cancelation from background click in `BaseBottomSheet` at all. But I did notice we are allowing the swipe to dismiss, which is causing issues with this flow. Should we disable the swipe behavior for the entire bottom sheet or keep it?

Reply:
I ended up disabling it since it's crashing the app when we dismiss the sheet and hit back button. My guess is probably because we disabled the cancelation of the dialog and then handling the back press ourselves.